test/mailers/precheck_mailer_test.rb

```diff
@@ -0,0 +1,4 @@
+require 'test_helper'
+
+class PrecheckMailerTest < ActionMailer::TestCase
+end
```

Comment:
Shall we add some tests?

Reply:
Added basic tests, one is commented out as the logic for a route it uses is on the controller PR 👍